libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -220,4 +254,59 @@ export class SkyAgGridWrapperComponent
       }
     }
   }
+
+  #updateInlineHelp(column: Column, context?: any): void {
+    const colId = column.getId();
+    const columnHeader = this.elementRef.nativeElement.querySelector(
+      `div.ag-header-cell[col-id="${colId}"] div.ag-cell-label-container`
+    ) as HTMLElement;
```

Comment:
Find the column header. The `div.ag-cell-label-container` seems stable but I did need to adjust the styles. The child of this element gets wiped by AG Grid so I wasn't able to use that.